drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -82,18 +86,14 @@ local function do_configure(driver, device)
   -- where devices with multiple endpoints with the same device type cannot be detected
   local switch_eps = device:get_endpoints(clusters.OnOff.ID)
   local num_switch_eps = #switch_eps
-  table.sort(switch_eps)
-  --Default MCD switch handling depends on consecutive endpoint numbering
-  if num_switch_eps == switch_eps[num_switch_eps] then
-    if num_switch_eps > 1 then
-      device:try_update_metadata({profile = string.format("switch-%d", math.min(num_switch_eps, MAX_MULTI_SWITCH_EPS))})
-    end
-    if num_switch_eps > MAX_MULTI_SWITCH_EPS then
-      error(string.format(
-        "Matter multi switch device will not function. Profile doesn't exist with %d components",
-        num_switch_eps
-      ))
-    end
+  if num_switch_eps > 1 then
+    device:try_update_metadata({profile = string.format("switch-%d", math.min(num_switch_eps, MAX_MULTI_SWITCH_EPS))})
```

Comment:
As it was used as such in the past,  it could have been useful to define `switch-%d` as a constant, to be shared across all drivers, to ensure the format would be consistent across.